kornia/color/rgb.py

```diff
@@ -2,6 +2,52 @@
 import torch.nn as nn
 
 
+class RgbToRgba(nn.Module):
+    r"""Convert image from RGB to RGBA.
+
+    Add an alpha channel to existing image of special value named alpha_val
+
+    args:
+    image (torch.Tensor): A Three Channeled Image Tensor
+    alpha_val (int): An integer between 0 to 255
+    returns:
+            torch.Tensor: RGBA version of the image.
+
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 4, H, W)`
+
+    Examples::
+
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> rgba = kornia.color.RgbToRgba(alpha_val)
+        >>> output = rgba(input)  # 2x4x4x5
+
+   """
+    def __init__(self) -> None:
+
+        super(RgbToRgba, self).__init__()
+
+    def forward(self, image: torch.Tensor, alpha_val: float) -> torch.Tensor:
+        return rgb_to_rgba(image, alpha_val)
+
+
+def rgb_to_rgba(image: torch.Tensor, alpha_val: float):
```

Comment:
return type

Reply:
not mandatory now nut should be able to support a tensor as alpha values too. Maybe for a next PR or add a `# TODO` note here.